Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:40
+FROM fedora:41
```

Comment:
LGTM but the build is failing: https://github.com/loopholelabs/linux-pvm-ci/actions/runs/11646997347/job/32577013246?pr=9#step:7:254

Reply:
Hmm, it looks like there have been changes to the group system in F41 (since it switches to `dnf5`). I'll take a look!